activemodel/CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
+*   Fix validation callbacks on multiple context
+
+    *Yoshiyuki Hirano*
+
 ## Rails 5.2.0.beta2 (November 28, 2017) ##
 
 *   No changes.
 
-
```

Comment:
The two blank lines are intended. Please don' touch it.

Reply:
Sorry, I understood. I've fixed it.